### PR TITLE
DEVDOCS-3933: [net new] Stencil, adding wallet buttons

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-3933]

## What changed?
Created a new article explaining how to add wallet buttons on the PDP.

## Anything else?
[TOC update 167](https://github.com/bigcommerce-labs/next-dev-center-prototype/pull/167)

[DEVDOCS-3933]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3933?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/stencil-docs/deploying-a-theme/wallet-buttons.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+}
+```
+
+<!-- theme: info -->
+> #### Default number of buttons
+> If the `paymentbuttons-number-of-buttons` property is not defined, only **one** button will be shown. The customer will have access to any other buttons by clicking **More payment options**.
+
+2. To add a **Page Builder** dialog for the merchant to configure the number of visible wallet buttons, add the following to the **Payments** section of `schema.json`:
+
+```json title="schema.json, checkbox configuration"
```
Comment (slsriehl) [inline docs/stencil-docs/deploying-a-theme/wallet-buttons.md:78]: I messed up here, this should say something about visible wallet buttons.

File: docs/stencil-docs/deploying-a-theme/wallet-buttons.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+}
+```
+
+The button should look as follows:
+
+![wallet-buttons-02](https://storage.googleapis.com/bigcommerce-production-dev-center/images/wallet-buttons-02.png "wallet-buttons-quantity")
+
+## Adding sorting for the wallet buttons
+
+Merchants can choose an order for rendering wallet buttons. To add a sorting feature, perform the following steps.
```
Comment (slsriehl) [inline docs/stencil-docs/deploying-a-theme/wallet-buttons.md:117]:
```suggestion
Merchants can choose an order for rendering wallet buttons. To add a sorting feature, perform the following steps:
```

**Review by stanzikratelbc — DISMISSED** [review]
LGTM

**Review by stanzikratelbc — APPROVED** [review]
LGTM

**Review by slsriehl — APPROVED** [review]
I have questions about the add a section heading example and callout, but as we have tickets to amend this already in the mix, I'm happy to publish now.

**Comment by bc-traciporter** [comment]
Thank you @slsriehl

**Pull request merged**